app/services/guest-cam/guest-track.ts

```diff
@@ -58,18 +58,19 @@ export class GuestTrack extends MediasoupEntity {
     if (this.opts.kind !== 'video') return;
     if (!this.consumerId) return;
 
-    const totalStreams = this.guestCamService.consumer.guests.length;
+    const height = this.guestCamService.views.getSourceForGuest(this.opts.streamId).height;
```

Comment:
This may not be what you actually want.  This is the size of the actual underlying video feed, not the size of the actual scene item in the scene.

Reply:
oh got it, is there a way to access that?

Reply:
It will be kind of a complex operation.  You'll need to find all the scene items referencing a particular source, find the one with the largest scale, and then multiply that scale by the original heigh of the source.